src/hardware/disney.cpp

```diff
@@ -28,6 +28,8 @@
 
 #define DISNEY_SIZE 128
 
+enum STATE { IDLE, RUNNING, FINISHED, ANALYZING };
```

Comment:
*Suggestion only*:

`STATE` is very generic name… maybe too generic. Ho about `enum class DS_STATE`? I assume DS in this context means "Disney Sound" and not "DOSBox"

If you'll end up going this route, I think new commit will be appropriate (to avoid unnecessary conflicts with other work in this PR).

Reply:
>  I think new commit will be appropriate (to avoid unnecessary conflicts with other work in this PR).

:+1:  I know sometimes this mixing-of-prior changes (unrelated to a variable rename) has made ammending prior commits quite painful. A separate commit will be clean and easy.